readthedocs/projects/views/private.py

```diff
@@ -80,9 +81,24 @@ class ProjectDashboard(PrivateViewMixin, ListView):
     model = Project
     template_name = 'projects/project_dashboard.html'
 
+    def email_notification(self, request):
+        email_qs = request.user.emailaddress_set.filter(primary=True)
+        email = email_qs.first()
+        if email:
```

Comment:
Maybe we can write this like

    if not email or not email.verified:
        # do the notification

this way, we do not repeat the line that creates the notification.

Reply:
Thanks for the advice.